packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { string, element, oneOf, shape, arrayOf } from 'prop-types';
+import styled from '@emotion/styled';
+import { C_LUNAR, C_EBON, C_METAL } from '@bbc/psammead-styles/colours';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getBrevier } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+const ltrRtl = (ltrValue, rtlValue) => ({ dir }) =>
+  dir === 'ltr' ? ltrValue : rtlValue;
+
+const topicTagParent = `
+  word-break: break-word;
+  & a {
+    display: flex;
+    padding-top: ${GEL_SPACING};
+    padding-bottom: ${GEL_SPACING};
+    text-decoration: none;
+    color: ${C_EBON};
+
+    &:hover {
+      text-decoration: underline;
+    }
+    &:visited {
+      color: ${C_METAL};
+    }
+  }
+
+  & a > span {
+    background-color: ${C_LUNAR};
+    padding: 0.375rem 0.4375rem;
+  }
+
+  &:not(:first-of-type) {
+    margin-top: 0.0625rem;
+  }
+`;
+
+const topicTagsContainer = `
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+`;
+
+const SingleTopicTagContainer = styled.div`
+  ${topicTagsContainer}
+`;
+
+const TopicsList = styled.ul`
+  list-style-type: none;
+  ${topicTagsContainer}
+`;
+
+const TopicTagListItem = styled.li`
+  ${({ service }) => getSansRegular(service)}
+  ${({ script }) => script && getBrevier(script)}
+  ${ltrRtl('margin-right', 'margin-left')}: ${GEL_SPACING};
+
+  ${topicTagParent}
+`;
+
+const SingleTopicTagItem = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${({ script }) => script && getBrevier(script)}
+  ${ltrRtl('margin-right', 'margin-left')}: ${GEL_SPACING};
+
+  ${topicTagParent}
+`;
+
+export const TopicTag = ({ topicName, topicLink }) => (
+  <a href={topicLink}>
+    <span>{topicName}</span>
```

Comment:
Do we need the `span` here? Or can we just have the topicName in the `a` tag?

Reply:
Think this is to achieve the required height for the touch target area along with the design requirements

Reply:
The `span` here is essentially the grey box of the `TopicTag` component, and was used so that the hit area of the link can be expanded beyond the borders of the grey box, to a minimum of `44px` 👍.